avro4s-json/src/main/scala/com/sksamuel/avro4s/json/JsonToAvroConverter.scala

```diff
@@ -38,7 +38,9 @@ class JsonToAvroConverter(namespace: String, avroStringTypeIsString: Boolean = f
     case JString(_) => createStringSchema
     case JObject(values) =>
       val record = Schema.createRecord(name, null, namespace, false)
-      val fields = values.map { case (name, value) => new Schema.Field(name, convert(name, value), null, null) }
+      val doc: String = null
+      val default: AnyRef = null
+      val fields = values.map { case (name, value) => new Schema.Field(name, convert(name, value), doc, default) }
```

Comment:
Some `Schema.Field` constructors were deprecated. Providing explicit types for the arguments ensures the correct non-deprecated constructor is called.